content/joysticks/winwing/winwing-cdu/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: WINWING CDU
+weight: 50
+---
+
+WINWING CDU display support is automatic on run, if a supported plane and a CDU are detected. To prevent MobiFlight from taking over the CDU, disable it in `Extras->Settings->Peripherals`.
+
+## Installing Python
+
+Follow the [guide](/guides/installing-python/).
```

Comment:
This would be better for new users if it were combined with the later statement that it a pre-requisite, instead of just a header that says "Installing Python" with no context as to why this would be important.

Perhaps this can be restructured as a callout:

> [!IMPORTANT]
> WINWING CDU support depends on Python. See the [Python installation guide](/guides/installing-python) for steps on how to install Python for use with MobiFlight.